kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/ExpressionEditorViewImpl.java

```diff
@@ -522,7 +531,7 @@ private void updateCanvasNodeName() {
         return modelName -> {
             final EntryInfo[] parametersFromModel = pmmlDocumentMetadataProvider.getPMMLDocumentModelParameterNames(documentName, modelName)
                     .stream()
-                    .map(parameter -> new EntryInfo(parameter, UNDEFINED.getText()))
+                    .map(parameter -> new EntryInfo(parameter, parameter, UNDEFINED.getText()))
```

Comment:
@vpellegrino Can you please help me to understand this? I compared this assessment with the old editor, and I found this https://github.com/kiegroup/kogito-editors-java/blob/33ba86be4c395b9cae7396dce6c043352dfc111b/kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/expressions/types/function/supplementary/pmml/model/LiteralExpressionPMMLDocumentModelEditorDefinition.java#L198
So, should I consider `EntryInfo` class (new editor) equals to `InformationItem` class (old editor) ?  If true, I expect we should pass the same parameters here, what I'm missing here?

Reply:
Precisely, `EntryInfo` is the `props` class related to the `InformationItem` class in the `model`.
First of all, thanks for throwing this question.
Yeah, actually, in the case of the parameters, the only field used is the `name` field; while `id` and `dataType` are populated but never used.
So, in the old implementation, the id is generated `new Id()`, while dataType is `any`.
I will align the change with the same, but it should not impact any use case.